Source/Core/Core/IOS/DI/DI.cpp

```diff
@@ -166,19 +166,20 @@ std::optional<DI::DIResult> DI::StartIOCtl(const IOCtlRequest& request)
   {
     const u32 length = Memory::Read_U32(request.buffer_in + 4);
     const u32 position = Memory::Read_U32(request.buffer_in + 8);
-    INFO_LOG(IOS_DI, "DVDLowRead: offset 0x%08x (byte 0x%09" PRIx64 "), length 0x%x", position,
```

Comment:
Is the `#include <cinttypes>` still necessary with the removal of those macro uses?